### DISCUSSION TITLE
LF1 MyOpenMath

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#352] [General]
Reviewer: @AbbyANoble 

Questions 1 - 4: Find AROC from a table (some with context)
Questions 5 - 12: Find AROC from a graph. The first few show a graph with specific points (some with context) and then the graphs are just given and students are asked to find AROC for a given interval.
Questions 13 - 17: Fing AROC from an equation. Some say to find AROC from x= to x= and then a couple are given in interval notation.
Question 18: Find AROC from a story/word problem. I had a hard time finding different context problems without also having instantaneous rate of change in the question as well...
Questions 19 - 20: Identify positive/negative slopes and compare them.
Questions 21 - Determine rise, run, and slope.
Questions 23 - 31: Find slope from a graph. The first few have the points on the graph and then it moves to finding the slope of the line in general.
Questions 32 - 33: Interpretation of Slope (not sure we wanted these....I don't know if I did a good job in this section talking about the rates and units...)
Questions 34 - 40: Slope given two points.
Questions 41 - 43: Slope from a table.

**Comment by AbbyANoble** [comment]
Looks good to me! Kind of long, but a lot to choose from if someone wanted to pare it down.
